examples/resources/biganimal_cluster/single_node/bah_gcp/resource.tf

```diff
@@ -0,0 +1,75 @@
+terraform {
```

Comment:
No service account ids?

Reply:
I added those values as commented out and created https://github.com/EnterpriseDB/terraform-provider-biganimal/issues/339.